test/test_collection.py

```diff
@@ -390,26 +390,28 @@ def test_index_info(self):
 
         db.test.create_index("hello")
         self.assertEqual(len(db.test.index_information()), 2)
-        self.assertEqual(db.test.index_information()["hello_1"]["key"],
+        self.assertEqual(list(db.test.index_information()["hello_1"]["key"]),
```

Comment:
Why are these changes needed? The output of `index_information()` should not change in this PR.

Reply:
Those changes are needed to get the tests to pass, because otherwise it tries to compare a dict_items object with a list. I am not sure why the output of index_information is changing but it appears it is.

Reply:
Please investigate why `index_information` has changed. It should not be returning a dict_items object. We'll need to revert these test changes too.